doc/developers/tips.rst

```diff
@@ -90,6 +93,19 @@ When a unit test fails, the following tricks can make debugging easier:
 
          pytest --pdbcls=IPython.terminal.debugger:TerminalPdb --capture no
 
+Other `pytest` options that may become useful include:
+
+  - ``-x`` which exists on the first failed test
+  - ``--lf`` to rerun the tests that failed on the previous run
```

Comment:
Maybe also `--ff` to start by failed tests.

Reply:
Didn't know this one, thanks ^^